<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpursAPI</class>
 <widget class="QMainWindow" name="SpursAPI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpursAPI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="welcome_page"/>
      <widget class="QWidget" name="login_page"/>
      <widget class="QWidget" name="main_page"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
